pkgs/tools/system/augeas/default.nix

```
@@ -11,10 +11,12 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ readline libxml2 ];
 
+  enableParallelBuilding = true;
+
   meta = with lib; {
     description = "Configuration editing tool";
-    license = licenses.lgpl2;
-    homepage = "http://augeas.net/";
+    license = licenses.lgpl2Only;
```

[nit](http://augeas.net/faq.html#:~:text=Augeas%20is%20distributed,2.1`) [pick](https://github.com/hercules-team/augeas/blob/4f3bbeb3f7823c62292d3c273108a4dd75dfb1ab/COPYING#L3)

```suggestion
    license = licenses.lgpl21Only;
```